scripts/add_clog_to_manifest.ts

```diff
@@ -0,0 +1,96 @@
+/**
+ * @file add_clog_to_manifest.ts
+ * @description Adds a changelog entry to manifest.json
+ * @example add_clog_to_manifest.ts --version=1.18.3
+ */
+import { writeFileSync } from "fs";
```

Comment:
this is a nitpick but the style we use in product/coder is to skip trailing semicolons, can we copy over our prettierrc so that formatting is consistent? (assuming it actually fixes that)

no worries if not, this is perfectly cromulent as-is

Reply:
Sure thing, we can update 1 line in `.prettierrc` and run `fmt:fix` :D